CHANGES.rst

```diff
@@ -166,9 +166,9 @@ astropy.units
   ``bool(0 * u.dimensionless_unscaled)`` evaluates to ``True``. In the future,
   attempting to convert a ``Quantity`` to a ``bool`` will raise ``ValueError``.
   [#6580, #6590]
- - Modify the ``brightness_temperature`` equivalency to provide a surface
-   brightness equivalency instead of the awkward assumed-per-beam equivalency
-   that previously existed [#5173, #6663]
+- Modify the ``brightness_temperature`` equivalency to provide a surface
```

Comment:
It is the lack of a line between items that's the problem - and the rebase often does that, annoyingly.

Reply:
I'll sort out minor changelog issues like this before doing a release, so no need to hold back merging (on days like today).